extras/vfd16lf01/LICENSE

```diff
@@ -0,0 +1,7 @@
+This is a driver for Samsung 16LF01 series VFD.
+
+Originate written by Daniele Napolitano, distributed under GPL.
```

Comment:
If you can get permission from all of the original authors (in this case it looks like there's only one, which makes it easier) to distribute this under a more permissive license (i.e. BSD or MIT), then yeah, that would work..

If you do get that permission, you should change this bit to say something like "originally distributed under the GPL, but special permission given to license this code under BSD for esp-open-rtos"